app/services/member_services_member_service.rb

```diff
@@ -1,28 +0,0 @@
-# frozen_string_literal: true
-
-# It's a member service... for member services!
-class MemberServicesMemberService
-  attr_reader :member, :errors, :status
-
-  def initialize(params)
-    @params = params
-  end
-
-  def update
-    @member = Member.find_by_email(@params[:email])
-
-    unless @member
-      @errors = ["No member associated with email address #{@params[:email]}."]
-      @status = 404
-      return false
-    end
-
-    if @member.update_attributes(@params)
-      return true
-    else
-      @errors = ["Updating member details failed for #{@member.email}."]
```

Comment:
I would keep this in. It's nice to get a specific message when there is an issue updating attributes, rather than getting a random 500 that'll take a good while to debug.

Reply:
If there's a problem with the update, the api will return a 422. If you're debugging this case all the information is already in the standard logs (response status, params) , so logging this is redundant.